src/wallet/wallet.ts

```diff
@@ -135,6 +138,40 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    **/
   clearSensitiveData() {
     this.seed = null;
+    this.authXpriv = null;
+  }
+
+  /**
+   * Get auth xpubkey in the 280' purpose path from seed
+   *
+   * @param {String} seed 24 words
+   * @param {Object} options Options with passphrase and networkName
+   *
+   * @return {String} Wallet xpubkey
```

Comment:
```suggestion
   * @return {String} auth xpubkey
```

Reply:
Thanks! Fixed in 7be2bddfaa7280addf82144f2b3ef67767fac655